tests/functional/views/lti/basic_lti_launch_test.py

```diff
@@ -23,33 +21,35 @@ def test_requests_with_no_oauth_signature_are_forbidden(
         assert response.headers["Content-Type"] == Any.string.matching("^text/html")
         assert response.html
 
-    def test_unconfigured_basic_lti_launch(self, lti_params, do_lti_launch):
+    def test_unconfigured_basic_lti_launch(
+        self, lti_params, do_lti_launch, get_client_config
```

Comment:
A few changes to accommodate get_client_config being now a fixture instead of a local function.